libbindgen/src/ir/annotations.rs

```diff
@@ -39,6 +39,25 @@ pub struct Annotations {
     /// The kind of accessor this field will have. Also can be applied to
     /// structs so all the fields inside share it by default.
     accessor_kind: Option<FieldAccessorKind>,
+    /// Whether this enum variant should be constified.
+    ///
+    /// This is controlled by the `constant` attribute, this way:
+    ///
+    /// ```cpp
+    /// enum Foo {
+    ///     Bar = 0, /**< <div rustbindgen constant></div> */
+    ///     Baz = 0,
+    /// };
+    /// ```
+    ///
+    /// In that case, bindgen will generate a constant for `Bar` instead of
+    /// `Baz`.
+    ///
+    /// You can see the kind of comments that are accepted in the Doxygen
+    /// documentation:
+    ///
+    /// http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html
```

Comment:
The last paragraph as well as the link doesn't seem to be something specific to this field. Probably this is comment which should be added to the `Annotation` struct.

Also, `hide` now supports enum variants in addition to types, which should be mentioned in the document of that field.